RangeBarSample/src/main/res/layout/activity_recyclerview.xml

```diff
@@ -0,0 +1,23 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="match_parent"
+        android:layout_height="130dp"
+        android:background="@color/indigo500"
+        android:gravity="center"
+        android:text="@string/title"
+        android:textColor="@color/white"
```

Comment:
No need for title block. the Ui looks bad. Instead rename the activity title to reflect what activity does. Activity title could be renamed to `Rangebar RecyclerVIew Sample`

Reply:
Done.